ast/gen.go

```diff
@@ -0,0 +1,4 @@
+package ast
+
+//go:generate flatc --go ./ast.fbs
```

Comment:
Is there any way you could generate these files to an internal package? So that the files go to the package: `ast/internal/fbast`.